app/services/stake_pools/calculate_average_stats.rb

```diff
@@ -33,6 +36,10 @@ def call
 
     attr_reader :pool, :validator, :validator_ids, :sp_attrs
 
+    def validator_delinquent?(score)
```

Comment:
This new name is not much better. This method is NOT checking if validator is delinquent (delinquent attribute stores that information). This method is checking two conditions, just one of them happens to be related to deliquency.
Should be called something like `delinquent_with_minimum_stake` or similar.